src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -288,6 +293,11 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         return _commandline;
     }
 
+    winrt::hstring ConptyConnection::Title() const
```

Comment:
The name of this property implies that you could call it at any time and get the current title, which isn't totally true -- we may want to make it more clear that it's a one-time thing only to be used in certain circumstances